keyboards/kbdfans/kbd67/rev2/keymaps/ansi_split_space/config.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2018 MechMerlin
```

Comment:
This file can be deleted.

Reply:
Yeah, I thought it was odd but just sort of blindly following the pattern.